extension/src/experiments/model/index.ts

```diff
@@ -117,6 +118,21 @@ export class ExperimentsModel extends ModelWithPersistence {
     this.setColoredStatus(runningExperiments)
   }
 
+  public transformAndSet_(
```

Comment:
`_` is the key here. Every time I have to change the function signature/test data I'll duplicate and add a `_`. This saves me trying to fix absolutely everything in one massive hit (head would explode). I'll be taking notes on everything that I see that needs to be updated downstream and refraining from stacking those requirements/follow-ups up until after the breaking change.

Reply:
This is the bad news: 

![image](https://user-images.githubusercontent.com/37993418/231061708-2c597601-4de3-4b8b-ae18-024e89eefc5f.png)